File: Python/Plane.py
class Plane:

    # Constructor
    def __init__(self, fuel, numPass, timezone):
        self._fuel = fuel               # Amount of fuel
        self._numPass = numPass         # Number of pass
        self._timezone = timezone       # The timezone the plane is in
    
    # Recharge the Fuel
    def incFuel(self, x):
        self_fuel += x
   
    # Decrease the Fuel by x
    def decFuel(self, x):
        self._fuel -= x

    # Increase numPpass by x
    def incPass(self, x):
        self._numPass += x
    
    # Decrease numPass by x
    def decPass(self, x):
        self._numPasss -= x
    
    
    # Increase timeZone by x
    def incTimezone(self, x):
        self._timezone += x

    # Decrease timeZone by x
    def decTimezone(self, x):
        self._timezone -= x

    

File: c#/Plane.cs
using System;

public class Plane
{
    private double _fuel;
    private int _numPass, _timezone;
    private List<Passenger> pass = new List<Passenger>; 

    // Constructor
    
    public Plane(double fuel, int numPass, int timezone)
    {
        _fuel = fuel;
        _numPass = numPass;
        _timezone = timezone;
    }
    
    // Return the variable
    public double Fuel
    {
     get{return _fuel;}
    }

    public int NumPass
    {
     get{return _numPass;}
    }
    public int Timezone
    {
     get{return _timezone;}
    }

    public Passenger[] Pass
    {
        get{return _pass;}
    }

    // Increase fuel by x
    public void incFuel(double x)
    {
        _fuel += x; 
    }
    
    // Decrease fuel by x
    public void decFuel(double x)
    {
        _fuel -= x; 
    }

    // Add a new passenger to the plane
    public void addPass(Passenger x)
    {
        pass.Add(x);
       _numPass += 1;
    }
    
    // Remove a passenger from the plane
    public void removePass(Passenger x)
    {
        pass.Remove(new Passenger(){passengerID = x.PassengerID});
        _numPass -= 1;
    }

    // Increase timezone by x
    public void incTimezone(int x)
    {
        _timezone += x; 
    }

    // Decrease timezone by x
    public void decTimezone(int x)
    {
        _timezone -= x; 
    }

    public bool full()
    {
        if (_numPass==10)
            return false;
        else
            return true;
    }
}
